ada-project-docs/wave_04.md

```diff
@@ -0,0 +1,47 @@
+# OPTIONAL Wave 4: Enhancements
```

Comment:
It looks like the base README is unchanged, it would be helpful to remove "Optional" from the Wave 3 description and add a new link for this file as the new optional enhancements.